core/tests/protractor_desktop/additionalEditorFeatures.js

```diff
@@ -383,6 +385,390 @@ describe('Full exploration editor', function() {
     await users.logout();
   });
 
+  it(
+    'should show merge changes when the changes are not conflicting',
```

Comment:
What do you mean by "show merge changes" here? I'm not able to understand the description here.

Reply:
Changed the description.